src/Frontend/Core/Layout/Templates/Pagination.html.twig

```diff
@@ -16,7 +16,7 @@
             {% if pagination.first %}
               {% for pagination in pagination.first %}
```

Comment:
Aren't these for loops the cause of the problem? There can only be one first and last item.
I think you can just drop the loops and use `{{ pagination.first.url }}` for example.

Can you try this @GhaziTriki ?